Require/GlobalIsDefined/module.js

```diff
@@ -0,0 +1,5 @@
+module.exports = { 
+	accessGlobalObject: function(s) {
+		TNSLog(global);
```

Comment:
`TNSLog` accepts a string, so our runtime crashes when given an object. We have logged it here https://github.com/NativeScript/ios-runtime/issues/251, but it isn't ready yet.